### ISSUE TITLE
Theme classes broken

### ISSUE THREAD

**Issue by mbertogliati** [ethanblake4/flutter_eval#86]
Hi, I've been trying to run the "Quickstart for dynamic execution and server-driven UI" from the /readme.md.
I get the following error when trying to build the app:

```
../../../../../AppData/Local/Pub/Cache/hosted/pub.dev/flutter_eval-0.7.5/lib/src/material/text_theme.dart(108,7): error GC6690633: No named parameter with the name 'headline1'. [C:\flutter\build\windows\x64\flutter\flutter_assemble.vcxproj]
../../../../../AppData/Local/Pub/Cache/hosted/pub.dev/flutter_eval-0.7.5/lib/src/material/text_theme.dart(195,34): error G4127D1E8: The getter 'headline1' isn't defined for the class 'TextTheme'. [C:\flutter\build\windows\x64\flutter\flutter_assemble.vcxproj]
../../../../../AppData/Local/Pub/Cache/hosted/pub.dev/flutter_eval-0.7.5/lib/src/material/text_theme.dart(198,34): error G4127D1E8: The getter 'headline2' isn't defined for the class 'TextTheme'. [C:\flutter\build\windows\x64\flutter\flutter_assemble.vcxproj]
../../../../../AppData/Local/Pub/Cache/hosted/pub.dev/flutter_eval-0.7.5/lib/src/material/text_theme.dart(201,34): error G4127D1E8: The getter 'headline3' isn't defined for the class 'TextTheme'. [C:\flutter\build\windows\x64\flutter\flutter_assemble.vcxproj]
../../../../../AppData/Local/Pub/Cache/hosted/pub.dev/flutter_eval-0.7.5/lib/src/material/text_theme.dart(204,34): error G4127D1E8: The getter 'headline4' isn't defined for the class 'TextTheme'. [C:\flutter\build\windows\x64\flutter\flutter_assemble.vcxproj]
../../../../../AppData/Local/Pub/Cache/hosted/pub.dev/flutter_eval-0.7.5/lib/src/material/text_theme.dart(207,34): error G4127D1E8: The getter 'headline5' isn't defined for the class 'TextTheme'. [C:\flutter\build\windows\x64\flutter\flutter_assemble.vcxproj]
../../../../../AppData/Local/Pub/Cache/hosted/pub.dev/flutter_eval-0.7.5/lib/src/material/text_theme.dart(210,34): error G4127D1E8: The getter 'headline6' isn't defined for the class 'TextTheme'. [C:\flutter\build\windows\x64\flutter\flutter_assemble.vcxproj]
../../../../../AppData/Local/Pub/Cache/hosted/pub.dev/flutter_eval-0.7.5/lib/src/material/text_theme.dart(213,34): error G4127D1E8: The getter 'subtitle1' isn't defined for the class 'TextTheme'. [C:\flutter\build\windows\x64\flutter\flutter_assemble.vcxproj]
../../../../../AppData/Local/Pub/Cache/hosted/pub.dev/flutter_eval-0.7.5/lib/src/material/text_theme.dart(216,34): error G4127D1E8: The getter 'subtitle2' isn't defined for the class 'TextTheme'. [C:\flutter\build\windows\x64\flutter\flutter_assemble.vcxproj]
../../../../../AppData/Local/Pub/Cache/hosted/pub.dev/flutter_eval-0.7.5/lib/src/material/text_theme.dart(219,34): error G4127D1E8: The getter 'bodyText1' isn't defined for the class 'TextTheme'. [C:\flutter\build\windows\x64\flutter\flutter_assemble.vcxproj]
../../../../../AppData/Local/Pub/Cache/hosted/pub.dev/flutter_eval-0.7.5/lib/src/material/text_theme.dart(222,34): error G4127D1E8: The getter 'bodyText2' isn't defined for the class 'TextTheme'. [C:\flutter\build\windows\x64\flutter\flutter_assemble.vcxproj]
../../../../../AppData/Local/Pub/Cache/hosted/pub.dev/flutter_eval-0.7.5/lib/src/material/theme_data.dart(122,7): error GC6690633: No named parameter with the name 'bottomAppBarColor'. [C:\flutter\build\windows\x64\flutter\flutter_assemble.vcxproj]
../../../../../AppData/Local/Pub/Cache/hosted/pub.dev/flutter_eval-0.7.5/lib/src/material/theme_data.dart(158,35): error G4127D1E8: The getter 'bottomAppBarColor' isn't defined for the class 'ThemeData'. [C:\flutter\build\windows\x64\flutter\flutter_assemble.vcxproj]
````
I believe it's related to this: https://docs.flutter.dev/release/breaking-changes/3-19-deprecations
I'm using Flutter v3.22

**Comment by wrbl606** [comment]
Fix is on [it's way](https://github.com/ethanblake4/flutter_eval/issues/83)

**Comment by ethanblake4** [comment]
Fixed in v0.7.6
